src/components/table/table.scss

```diff
@@ -99,7 +99,7 @@ $table--limel-table--row-selector: 1;
 
 .tabulator-row {
     transition: background-color 0.2s ease;
-    min-height: 2.25rem;
+    height: 2.25rem;
```

Comment:
I will drop this commit, since it's not needed for this PR, and I am worried that it creates more mess in future. It's a little bit of a breaking change.

For the records, I just keep this note here 👇 :

````
fix(table): avoid row height change

Sometimes, re-renderings after the initial render, or
changes within components in the table cells, or
clicking on rows, or size of the components in the
table cells would cause a resize on all rows in a table.
This always resulted in reduced quality of the experience.
This fix might break some carelessly designed UIs of
custom components used in table views.
But since we have so many issues with this,
and fixing other issues relies on having a fixed row height,
consumers have to adapt.
```